store/mockstore/mocktikv/cop_handler_dag.go

```diff
@@ -628,9 +627,6 @@ func (h *rpcHandler) fillUpData4SelectResponse(selResp *tipb.SelectResponse, dag
 	case tipb.EncodeType_TypeDefault:
 		h.encodeDefault(selResp, rows, dagReq.OutputOffsets)
 	case tipb.EncodeType_TypeChunk:
-		if dagReq.GetChunkMemoryLayout().GetEndian() != distsql.GetSystemEndian() {
```

Comment:
Is this check removed on purpose?

Reply:
To break the import circle. The check is not necessary.

Reply:
@wshwsh12 PTAL

Reply:
We can remove the check.  Mocktikv is always running with TiDB in the same machine.